jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -2522,6 +2522,11 @@ public boolean startsWith(String prefix) {
 	 *				when prefix is null
 	 */
 	public boolean startsWith(String prefix, int start) {
+                if (prefix.length()==1) {
+                   if (start < 0 || start>=this.length())
+                      return false;
+                   return charAtInternal(start) == prefix.charAtInternal(0);
+                }
```

Comment:
Please use tabs instead of spaces in JCL files. Also the String.java file is split into a Java 8 and Java 9+ versions. Can we make the change on both sides?

Reply:
Also please use braces ({}) for the body of the inner `if` statement and consistently use whitespace around all binary operators.